chanjo/sex/core.py

```diff
@@ -23,15 +23,19 @@ def predict_sex(x_coverage, y_coverage):
     Returns:
         str: prediction, ['male', 'female', 'unknown']
     """
-    # algoritm doesn't work if coverage is missing for X chromosome
-    if x_coverage == 0:
-        return 'unknown'
-    elif (y_coverage > 0) and (x_coverage / y_coverage < 10):
-        # this is the entire prediction, it's usually very obvious
+    # algorithm doesn't work if coverage is missing for X chromosome
+    if y_coverage == 0:
```

Comment:
Does not all samples disregarding sex have some coverage on both X and Y? In that case we could always return `unknown` if coverage is missing?